_tools/check_docs.ts

```diff
@@ -23,131 +23,38 @@ import {
-  "../fmt/bytes.ts",
-  "../fmt/colors.ts",
-  "../fmt/duration.ts",
-  "../fmt/printf.ts",
-  "../front_matter/mod.ts",
-  "../front_matter/unstable_yaml.ts",
-  "../fs/mod.ts",
-  "../fs/unstable_chmod.ts",
-  "../fs/unstable_copy_file.ts",
-  "../fs/unstable_link.ts",
-  "../fs/unstable_lstat.ts",
-  "../fs/unstable_make_temp_dir.ts",
-  "../fs/unstable_make_temp_file.ts",
-  "../fs/unstable_mkdir.ts",
-  "../fs/unstable_read_dir.ts",
-  "../fs/unstable_read_file.ts",
-  "../fs/unstable_read_link.ts",
-  "../fs/unstable_read_text_file.ts",
-  "../fs/unstable_real_path.ts",
-  "../fs/unstable_remove.ts",
-  "../fs/unstable_rename.ts",
-  "../fs/unstable_stat.ts",
-  "../fs/unstable_symlink.ts",
-  "../fs/unstable_truncate.ts",
-  "../fs/unstable_types.ts",
-  "../fs/unstable_umask.ts",
-  "../fs/unstable_utime.ts",
-  "../fs/unstable_write_file.ts",
-  "../fs/unstable_write_text_file.ts",
-  "../html/mod.ts",
-  "../html/unstable_is_valid_custom_element_name.ts",
-  "../http/mod.ts",
-  "../http/unstable_header.ts",
-  "../http/unstable_method.ts",
-  "../http/unstable_signed_cookie.ts",
-  "../ini/mod.ts",
-  "../internal/mod.ts",
-  "../io/mod.ts",
-  "../json/mod.ts",
-  "../jsonc/mod.ts",
-  "../log/base_handler.ts",
-  "../log/file_handler.ts",
-  "../log/warn.ts",
-  "../log/critical.ts",
-  "../log/debug.ts",
-  "../log/error.ts",
-  "../log/info.ts",
-  "../log/console_handler.ts",
-  "../log/formatters.ts",
-  "../log/get_logger.ts",
-  "../log/logger.ts",
-  "../media_types/mod.ts",
-  "../msgpack/mod.ts",
-  "../net/mod.ts",
-  "../net/unstable_get_network_address.ts",
-  "../path/mod.ts",
-  "../path/unstable_basename.ts",
-  "../path/unstable_dirname.ts",
-  "../path/unstable_extname.ts",
-  "../path/unstable_join.ts",
-  "../path/unstable_normalize.ts",
-  "../path/posix/mod.ts",
-  "../path/windows/mod.ts",
-  "../random/mod.ts",
-  "../regexp/mod.ts",
-  "../semver/mod.ts",
-  "../streams/mod.ts",
-  "../streams/unstable_fixed_chunk_stream.ts",
-  "../streams/unstable_to_lines.ts",
-  "../streams/unstable_to_bytes.ts",
-  "../tar/mod.ts",
-  "../text/mod.ts",
-  "../text/unstable_slugify.ts",
-  "../text/unstable_to_constant_case.ts",
-  "../testing/bdd.ts",
-  "../testing/mock.ts",
-  "../testing/snapshot.ts",
-  "../testing/time.ts",
-  "../testing/types.ts",
-  "../toml/mod.ts",
-  "../ulid/mod.ts",
-  "../uuid/mod.ts",
-  "../uuid/unstable_v7.ts",
-  "../webgpu/mod.ts",
-  "../yaml/mod.ts",
-] as const;
+const STABLE_VERSION = parse("1.0.0");
+const ROOT = new URL("../", import.meta.url);
+const ENTRY_POINT_URLS = [];
+for await (
+  const { path } of walk(ROOT, { exts: [".json"], match: [/deno.json$/] })
+) {
+  if (isTestFile(path)) continue;
+  const { version, exports } = JSON.parse(await Deno.readTextFile(path));
+  if (!version || !greaterOrEqual(parse(version), STABLE_VERSION)) continue;
+  for (const relativeFilePath of Object.values<string>(exports)) {
+    if (!relativeFilePath.endsWith(".ts")) continue;
```

Comment:
Is this needed? Are there any exports which export something besides a `.ts` file?

Reply:
Yes, `@std/html` exports a `json`: `"./named-entity-list.json": "./named_entity_list.json"`